### PR TITLE
EASY-1238 - replace SurrogateRelation from Metadata to Profile

### PR DESCRIPTION
**Pull request by rvanheest**
contributes to EASY-1238

#### When applied it will
* ~~replace the surrogate relation from the ddm metadata to the ddm profile~~
* [x] change the scheme in `AddDatasetMetadataToDeposit#126` to the new one ~~(waiting on result of xsd change by @lindareijnhoudt)~~
* [x] adjust the example results accordingly ~~(waiting on #54 to be merged)~~
* [x] fix some things I forgot to add in the examples in #54

#### Where should the reviewer @DANS-KNAW/easy start?
`AddDatasetMetadataToDeposit` and `AddDatasetMetadataToDepositSpec`

### CODE REVIEW

**Review by PaulBoon — APPROVED** [review]

File: src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala [hunk 1]
```diff
@@ -560,6 +561,27 @@ class AddDatasetMetadataToDepositSpec extends UnitSpec with BeforeAndAfterAll {
...
 
+  it should "return the expected xml with streaming surrogate if it is an A/V dataset" in {
+    object XmlTransform extends RewriteRule {
+      override def transform(n: Node): Seq[Node] =
+        n match {
+          case Elem(prefix, "profile", attribs, scope, children @ _*) =>
+            Elem(prefix, "profile", attribs, scope, false, children ++
+              <ddm:relation scheme="STREAMING_SURROGATE_RELATION">
+                /domain/randomdomainname/user/randomusername/collection/randomcollectionname/presentation/$presentation-placeholder
+              </ddm:relation> :_*)
```
Comment (jo-pol) [inline src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala]: shouldn't this be a one liner?

Comment (rvanheest) [reply src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala]: What do you mean? The XML element? Doesn't really matter. I'm not testing for formatting. That's removed in the test below.

File: src/test/resources/allfields/input/instructions.csv [hunk 2]
```diff
@@ -2,9 +2,9 @@ DATASET,DC_TITLE,DCT_ALTERNATIVE,DCX_CREATOR_TITLES,DCX_CREATOR_INITIALS,DCX_CRE
...
-ruimtereis02,Hubble ruimtetelescoop,,Dr,C,,Sagan,,Cornell University,,,,,,,,paul,,"De Hubble-ruimtetelescoop (Hubble Space Telescope, HST) bestaat uit een aantal precisie-instrumenten voor astronomische waarnemingen. Hij is genoemd naar de Amerikaanse astronoom Edwin Hubble, en draait sinds de lancering door de NASA op 24 april 1990 als een kunstmaan rond de aarde.",astronomie,,2015-2016,,,,,,,,,,ds2,,barbapappa,,,image/jpeg,NL,http://scholar.google.com,OPEN_ACCESS,,D30000,,,,,,,,,user001
+ruimtereis02,Hubble ruimtetelescoop,,Dr,C,,Sagan,,Cornell University,,,,,,,,paul,,"De Hubble-ruimtetelescoop (Hubble Space Telescope, HST) bestaat uit een aantal precisie-instrumenten voor astronomische waarnemingen. Hij is genoemd naar de Amerikaanse astronoom Edwin Hubble, en draait sinds de lancering door de NASA op 24 april 1990 als een kunstmaan rond de aarde.",astronomie,,2015-2016,,,,,,,,,,ds2,,barbapappa,,,image/jpeg,NL,http://scholar.google.com,OPEN_ACCESS,2016-07-30,D30000,,,,,,,,,user001
```
Comment (rvanheest) [inline src/test/resources/allfields/input/instructions.csv]: note: this change is unrelated to this PR, but is a remainder of #54

File: src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/dataset.xml [hunk 3]
```diff
@@ -73,8 +73,8 @@ xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/md/ddm/ http://easy.dans.kn
     <dcx-gml:spatial>
       <boundedBy xmlns="http://www.opengis.net/gml">
         <Envelope srsName="http://www.opengis.net/def/crs/EPSG/0/28992">
-          <lowerCorner>1 3</lowerCorner>
-          <upperCorner>2 4</upperCorner>
+          <lowerCorner>4 2</lowerCorner>
```
Comment (rvanheest) [inline src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/dataset.xml:76]: note: this change is unrelated to this PR, but is a remainder of #51

**Pull request merged**